src/content/index.md

```diff
@@ -5,3 +5,5 @@ title: 'Posts'
 [/loc-digital-garden](/loc-digital-garden)
 
 [/terraform-101](/terraform-101)
+
+[/data-plumbing](/data-plumbing)
```

Comment:
**discussion:** very exciting to get another post :) 

As it currently stands, this change will add a route but there's no content found. You can see this on the deploy preview site: https://deploy-preview-33--ladies-of-code-digital-garden.netlify.app/data-plumbing


I  would also expect to see the mark down file with the content https://github.com/LadiesOfCodeGroupSessions/loc-digital-garden/tree/main/src/content

Is that going to be added later?

Reply:
Oh my gosh Charlotte, I completely missed the actual content! I've added it now. Will double check

Reply:
It seems like it's ok now? Sorry about that! It worked ok locally but I missed the new file in the commit. Love the deployment preview :D

Reply:
LGTM.